src/components/manager.ts

```diff
@@ -865,13 +866,33 @@ export class Manager {
         }
     }
 
+    private kpsewhichBibPath(bib: string): string | undefined {
+        const kpsewhich = vscode.workspace.getConfiguration('latex-workshop').get('kpsewhich.path') as string
+        try {
+            const kpsewhichReturn = cs.sync(kpsewhich, ['-format=.bib', bib])
```

Comment:
Add a log message before calling kpsewhich. 

Please consider a case that kpsewhich does not immediately return.

Reply:
What do you have in mind exactly here? Using an asynchronous call with a timeout?

Reply:
What I am trying to say is that we should add log messages before executing commands. If there is no log message when the command blocks, we have no hint to debug. This is not specific for kpsewhich.

Reply:
The `cs.sync` function accepts a timeout option. It may be safe to use this option whenever a child process is spawned  synchronously. What do you think of it?

Reply:
I do not have a specific opinion.